ocaml/src/cli_asd.ml

```diff
@@ -370,6 +370,83 @@ let asd_range_cmd =
   in
   asd_range_t, info
 
+let asd_range_validate hosts port transport tls_config asd_id first
+                       verbose unescape verify_checksum follow show_all =
+  let finc = true
+  and last = None in
```

Comment:
when passing in --follow you probably also want to be able to specify last?
or maybe allow --follow within a prefix?
(that way you could manually trigger verify of the keys for a certain namespace)